pkg/scheduler/ugm/group_tracker_test.go

```diff
@@ -265,6 +265,33 @@ func TestGTSetAndClearMaxLimits(t *testing.T) {
 	assert.Equal(t, si.EventRecord_UG_GROUP_LIMIT, eventSystem.Events[1].EventChangeDetail)
 }
 
+func TestGTCanRunApp(t *testing.T) {
+	manager := GetUserManager()
+	defer manager.ClearConfigLimits()
+	user := security.UserGroup{User: "test", Groups: []string{"test"}}
+	groupTracker := newGroupTracker(user.User, newUGMEvents(mock.NewEventSystemDisabled()))
+	manager.userWildCardLimitsConfig["root.parent"] = &LimitConfig{
+		maxResources: resources.NewResourceFromMap(map[string]resources.Quantity{
+			"cpu": 1000,
+		}),
+		maxApplications: 3,
+	} // manipulate map directly to avoid hierarchy creation
+
+	hierarchy1 := strings.Split(path1, configs.DOT)
```

Comment:
this test does not care for `child1`, so using `path5` is more suitable.

Reply:
see answer below